### PR TITLE
Initial version of sil.bcc-arab.upp_ptwl1

### PR DESCRIPTION
**Pull request by rmlockwood**

### CODE REVIEW

**Comment by keyman-server** [comment]
This pull request is from an external repo and will not automatically be built. The build must still be passed before it can be merged. Ask one of the team members to make a manual build of this PR.

File: release/sil/sil.bcc-Arab.upp_ptwl1/README.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+-----------
+
+Balochi generated from template
+
+Links
+-----
+
+Supported Platforms
+-------------------
+ * Windows
```
Comment (darcywong00) [inline release/sil/sil.bcc-Arab.upp_ptwl1/README.md]: Current, only the mobile platforms support lexical models. Maybe the desktop platforms came from the Developer template? But they should be removed.

**Comment by darcywong00** [comment]
Adding a note for testers -  I was able to load this model with the "Balochi, Southern (Arabic) -> Balochi Persian" keyboard

**Comment by rmlockwood** [comment]
Changed it to just Mobile.

**Comment by darcywong00** [comment]
btw, I'm waiting for the lexical model compiler update from keymanapp/keyman#2496 to appear tomorrow before approving this PR.

**Comment by darcywong00** [comment]
@rmlockwood can you rebase your branch off of upstream/master so your PR will get the updated lexical model compiler?

**Comment by darcywong00** [comment]
@rmlockwood , sorry for the inconvenience. 

It looks like the Keyman Developer project template created your folder `sil.bcc-Arab.upp_ptw11` and we'd like to avoid further file-casing conflicts with non-Windows users by lower-casing the folder to `sil.bcc-arab.upp_ptw11`.

Which do you prefer?
a) handling the rename yourself or
b) have us submit your lexical-model on a new PR with the lower-cased paths

From what I recall, trying to fix these things in Windows can be frustrating.

**Comment by rmlockwood** [comment]
You can go ahead and fix it. Thanks.

**Comment by rmlockwood** [comment]
I think I did it. Check to see if it looks right.

 

Thanks,

Ron

**Comment by darcywong00** [comment]
>  Check to see if it looks right.

Hmm, on GitHub, it seems you merged branch sil.bcc-Arab.upp_ptwl1  onto itself? 
I went ahead and cleaned up the filepaths in #66 so we can continue the review there.

**Pull request closed without merging**